scripts/nmcli-py.py

```diff
@@ -61,17 +75,30 @@ def main():
 					sys.stdout.write(str(i)+' ')
 					sys.stdout.flush()
 					time.sleep(1)
-				print()
+				print('')
```

Comment:
whats this print for?

Reply:
prints new line since after sleep is over. Not important really